graphalyzer-server/src/file_handler/os/OSWrapper.py

```diff
@@ -49,7 +49,7 @@ def getOSPathHeader(self):
 			return "c:/"
 		else:
 			self.logger.debug("OS is Linux. Returning path header '/'")
-			return "/"
```

Comment:
My only gripe is that this function essentially doesn't do what the debug log says anymore